packages/entity-example/src/adapters/InMemoryDatabaseAdapter.ts

```diff
@@ -34,13 +34,13 @@ class InMemoryDatabaseAdapter<T extends Record<string, any>> extends EntityDatab
   protected async fetchManyWhereInternalAsync(
     _queryInterface: any,
     _tableName: string,
-    tableColumns: readonly string[],
-    tableValueValues: (readonly any[])[],
+    tableColumnTuple: readonly string[],
```

Comment:
This should still be tableColumns. "Tuple" to me here means the same as "row". Come to think of it, the other parameter could just be named `tableRows` or `tableTuples`.

- Column: the name of a single column
- Columns: an array, where each item is a column name
- Row or tuple: an array of values, where each item is this row's value for a specific column
- Rows or tuples: an array of rows/tuples, hence a 2D array

"Row" to me has more of a tabular connotation than "tuple", which I think is appropriate here: we have parameters that mention "table" and "column". 

In contrast, if we were thinking more abstractly and separated from database tables, I'd say "fieldNames" and "fieldValues/fieldTuples/fieldValueTuples" would be more appropriate.

Reply:
`fieldNames`/`fieldValues` refer to the entity fields already (since field names can differ from column names, often in casing). `fetchManyWhere` uses `key`/`values` which were `fieldName`/`fieldValue` in pre-composite times, and then translates them to database column names and calls this abstract method `fetchManyWhereInternalAsync` with columns. I'll go with `tableTuples`.